spec/takeaway_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'takeaway'
+
+describe Order do
+
+  let(:menu) { [{ item: "Chips", price: 0.99 },
+              { item: "Ham Sandwhich", price: 1.49 },
+              { item: "Curry and Rice", price: 3.99 },
+              { item: "Fried Chicken", price: 2.79 },
+              { item: "Pizza", price: 2.99 }] 
+  }
+
+  describe 'current order' do
+    it 'has empty array, length 5' do
+      expect(subject.current_order).to eq([0, 0, 0, 0, 0])
+    end
+  end
+
+  describe '.menu' do
+
+    it 'is an array, length 5' do
+      expect(subject.menu.length).to eq 5
+
+    end
+
+    it 'has chips which cost £0.99' do
+      expect(subject.menu[0][:item]).to eq "Chips"
```

Comment:
do you remember "magic numbers" ? you can create constant and set them. Therefore if you want to change later, you don't need to search them inside the all methods.